packages/core/src/components/next/AttentionBox/__stories__/AttentionBox.stories.tsx

```diff
@@ -0,0 +1,357 @@
+import React, { useState, useCallback, useEffect } from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import { Info, Invite } from "@vibe/icons";
+import { Button } from "../../../Button";
+import { AttentionBox } from "..";
+import type { AttentionBoxProps } from "../AttentionBox.types";
+import { Flex } from "../../../Flex";
+import { Heading } from "../../../Heading";
+import { Text } from "../../../Text";
+import { DialogContentContainer } from "../../../DialogContentContainer";
+import { Search } from "../../../Search";
+import { Avatar } from "../../../Avatar";
+import { Icon } from "../../../Icon";
+import { Box } from "../../../Box";
+import { Skeleton } from "../../../Skeleton";
+import person from "./assets/person.png";
+import contentImage from "./assets/content-image.png";
+
+type Story = StoryObj<typeof AttentionBox>;
+
+export default {
+  title: "Components/AttentionBox [New]",
+  component: AttentionBox
+} satisfies Meta<typeof AttentionBox>;
+
+export const Overview: Story = {
+  render: (args: AttentionBoxProps) => (
+    <div style={{ maxWidth: 600 }}>
+      <AttentionBox {...args} />
+    </div>
+  ),
+  args: {
+    title: "Attention box title",
+    text: "This action will cause your team to lose access to the account until you use the correct SSO source.",
+    action: {
+      text: "Button",
+      onClick: () => alert("Button clicked")
```

Comment:
I'm not a big fan of popping alerts on clicks in stories, we can log them in storybook instead

Reply:
where logging them? in the console log?